src/common/VerificationUtils.h

```diff
@@ -17,6 +17,11 @@ class VerificationUtils {
     bool implies(PTRef, PTRef); // Check the result with an external solver
 
     bool verifyInterpolant(PTRef partA, PTRef partB, PTRef itp);
+
+    bool verifyInterpolantInternal(PTRef partA, PTRef partB, PTRef itp);
+
+private:
+    bool checkSubsetCondition(PTRef p1, PTRef p2);
```

Comment:
This is in general a good idea.  Maybe we should change the others to have the word `external` in them?

Reply:
Good idea, I will do that.